### DISCUSSION TITLE
Favicon lock/set

### DISCUSSION THREAD

**Discussion by gord51north** [outline/outline#6614] [Ideas]
While it is cool that the Favicon in the browser switches to the Icon selected for each page/collection it makes some things a bit harder.

When there are a lot of tabs open in the browser they get very small, such that just the Favicon is visible.
This leads to returning to tabs by memory, and with the favicon changing to whatever the current page has set it becomes very difficult to quickly identify which tab Outline is.

Also, when viewing a page or collection that doesn't have an icon set the favicon defaults back to the Org/Company favicon.

It would be nice to be able to lock the favicon to keep this visual clutter from happening.

They key reason for this is that I find it nearly useless to have a specific page's favicon show, the context of what I was last on is irrelevant. I want to return to Outline itself, not necessarily the last page I was editing.

**Comment by VincentSC** [comment]
Same here. I'd like to set one favicon for all of Outline, which is not the same as the logo.

**Reply by tommoor** [reply]
An option to separately configure the Favicon seems pretty reasonable

**Reply by VincentSC** [reply]
Would also solve my problem, so :+1:

**Reply by tmeuze** [reply]
I would like to ask that #7768 possibly be rolled into this request. I run multiple workspace instances, and it would be nice to distinguish visually.
